src/Components/Cart/OrderSummary.js

```diff
@@ -1,32 +1,43 @@
 import React, { useState } from "react";
 import { useStateProvider } from "../../Context/StateContext/state-provider";
-import { cartPriceCalculator } from "./utilities/cart-price-calculator";
+import { useOrderSummary } from "./utilities/useOrderSummary";
 
```

Comment:
- Make sure you can use `useStateProvider` as a context or not. because `useState` is already a state management and you are sending it as a context. 
(I am not completely sure, so you can take other opinions as well)